src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,262 @@
+        tools.clear_sssd_cache()
+        client_e = multihost.client[0].ip
+        for _ in range(12):
+            ssh1 = SSHClient(client_e, username="foo1@example1",
+                             password="Secret123")
+            ssh1.close()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "grep 'All available child slots are full, "
+                                   "queuing request' /var/log/sssd/*")
+
+    def test_0002_bz1209483(self, multihost, backupsssdconf):
+        """
+        :title: sssd does not work as expected when id provider
+         equal to proxy and auth provider equal to ldap bz1209483
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1209483
+        :id: 2fa4834c-b252-11ec-919d-845cf3eff344
+        :steps:
+          1. Adding the user in local
+          2. sssd does not work as expected when id provider equal to proxy
+        :expectedresults:
+          1. Should succeed
+          2. Should not succeed
+        """
+        client_e = multihost.client[0].ip
+        tools = sssdTools(multihost.client[0])
+        # sssd does not work as expected when id provider equal to proxy
+        # and auth provider equal to ldap bz1209483
+        execute_cmd(multihost, "systemctl stop nslcd.service")
+        execute_cmd(multihost, "systemctl stop sssd")
+        execute_cmd(multihost, "useradd -u 2001 foo2")
+        execute_cmd(multihost, "echo 'pam.d/         pam_ldap.conf' > /etc/pam")
+        services = {'filter_groups': 'root', 'filter_users': 'root'}
+        tools.sssd_conf('nss', services)
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'debug_level': '0xFFF0',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'files',
+                         'auth_provider': 'ldap',
+                         'chpass_provider': 'ldap',
+                         'cache_credentials': 'true',
+                         'use_fully_qualified_names': 'False'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        assert "foo2:*:2001:2001::/home/foo2:/bin/bash" \
+               in execute_cmd(multihost, "getent passwd -s "
+                                         "sss foo2").stdout_text
+        ssh1 = SSHClient(client_e, username="foo2", password="Secret123")
+        ssh1.close()
+        execute_cmd(multihost, "userdel -rf foo2")
+
+    def test_bz1368467(self, multihost, backupsssdconf,
+                       create_350_posix_users):
+        """
+        :title: sssd runs out of available child slots and
+         starts queuing requests in proxy mode bz1368467
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1368467
+        :id: 062813d8-ca05-11ec-ae81-845cf3eff344
+        :steps:
+            1. sssd runs out of available child
+            slots and starts queuing requests in proxy mode
```

Comment:
Indent this line to start from under sssd. But this is not a step of the test, it is the test title

Reply:
Done